app/src/components/Project/CreateProject.tsx

```diff
@@ -12,8 +12,7 @@ const CreateProject = (props: ICreateProjectProps) => {
   const [title, setTitle] = useState('')
   const [description, setDescription] = useState('')
   const [tags, setTags] = useState<string[]>([])
-  const [buttonColor, setButtonColor] = useState('#EFEFEF')
-  const [buttonActiv, setButtonActiv] = useState('disabled')
+  const [buttonDisabled, setButtonDisabled] = useState(true)
```

Comment:
remove this state and add `const isTitleEmpty =  title.length === 0`. And make the button active when `disabled={!isTitleEmpty}`